src/web3-adapter/eth_gasStation.ts

```diff
@@ -0,0 +1,29 @@
+import https from "https";
```

Comment:
`https` is part of the Node standard library, so if you use this then this API won't work when called from the frontend; specifically, frontend builds will fail when they try to run Webpack since they can't find anything in `node_modules` named "https".

You'll need to write code which works in both the browser and in Node. One way to do this is to bring in a library which works in both places. [fetch-ponyfill](https://github.com/qubyte/fetch-ponyfill) is used elsewhere in alchemy-web3 so that would be a good one to use. Other common ones common choices include [axios](https://github.com/axios/axios) or [isometric-fetch](https://github.com/matthew-andrews/isomorphic-fetch).